docs/DeveloperGuide.md

```diff
@@ -1,11 +1,461 @@
+
+Prerequisites:    
+1. At least one Sprint in the SprintList
+
+Implementation:
+1. UI receive user input
+1. Parser parse user input
+1. Execute ViewSprintCommand
+    1. Get Sprint from SprintList
+    1. UI output to user
+
+#### Add Task to Sprint
+
+Link: [AddSprintTaskCommand.java](/src/main/java/seedu/duke/command/sprint/AddSprintTaskCommand.java) 
+
+Users can add Tasks existing in the Project Backlog to the Sprint, indicating that the Tasks are to be worked on during the iteration. 
+
+Prerequisites:    
+1. At least one Sprint in the SprintList
+1. At least one Task in the ProjectBacklog
+
+Implementation:
+1. UI receive user input
+1. Parser parse user input
+1. Execute AddSprintTaskCommand
+    1. On Sprint 
+        1. Add Task ID into sprintTaskIds
+    1. On Task
+        1. Add Sprint Number into sprintAllocatedTo
+    1. UI output to user
+
+
+#### Remove Task from Sprint
+
+Link: [RemoveSprintTaskCommand.java](/src/main/java/seedu/duke/command/sprint/RemoveSprintTaskCommand.java) 
+
+Users can remove Tasks from Sprint, indicating that the Tasks are deemed to not be worked on during the iteration. 
+
+Prerequisites:    
+1. At least one Sprint in the SprintList
+1. At least one Task is added to the selected Sprint
+
+Implementation:
+1. UI receive user input
+1. Parser parse user input
+1. Execute RemoveSprintTaskCommand
+    1. On Sprint 
+        1. Remove Task ID from sprintTaskIds
+    1. On Task
+        1. Remove Sprint Number from sprintAllocatedTo
+    1. UI output to user
+
+#### Allocate Sprint Tasks to Members   
+
+Link: [AllocateSprintTaskCommand.java](/src/main/java/seedu/duke/command/sprint/AllocateSprintTaskCommand.java) 
+ 
+Users can allocate Sprint Tasks to Members, indicating that the Tasks are assigned to the selected member to work on during the iteration. 
+
+Prerequisites:    
+1. At least one Task is added to the selected Sprint
+1. At least one Member is added to the Project
+
+Implementation:
+1. UI receive user input
+1. Parser parse user input
+1. Execute AllocateSprintTaskCommand
+    1. On Task
+        1. Add Member ID into membersAllocatedTo
+    1. On Member
+        1. Add Task ID into allocatedTaskIds
+
+
+### Storage
+To make the data persistent and portable, JSON has been chosen as the format for data to be saved to a persistent storage such as storage drives, thumb drives and any other storage medium which stores the program. JSON is also **human-readable** which allows users to directly modify the data file easily. This can be useful in certain scenarios such as fixing the data file in the event of data corruption.
+
+#### Location
+![Figure X: Running the Jar](image/developerguide/savingdata1.png "Running the Jar")  
+_Figure X: Running the Jar_  
+![Figure X: Running in IDE](image/developerguide/savingdata2.png "Running in IDE")  
+_Figure X: Running in IDE_  
```

Comment:
I really love these figures added. Gives a really good visual representation on where the saves are held.